std_msgs/Header header
common/Uint8TimeStamped cdcu_park_workmode
common/Uint8TimeStamped cdcu_park_st
common/Uint8TimeStamped cdcu_park_errlevel